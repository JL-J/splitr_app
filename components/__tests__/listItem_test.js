import React from 'react';
import { View, Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import ListItem from '../listItem';

describe('<ListItem />', function() {
  it('renders the component correctly', () => {
    const snap = renderer.create(
      <ListItem />).toJSON();
    expect(snap).toMatchSnapshot();
  });

  it('renders one text input component', () => {
    const wrapper = shallow(<ListItem />);
    expect(wrapper.find(Text)).toHaveLength(1);
  });

});
