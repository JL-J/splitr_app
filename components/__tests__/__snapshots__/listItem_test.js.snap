// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> renders the component correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View>
    <Text />
  </View>
</View>
`;
